leapp/utils/report.py

```diff
@@ -2,7 +2,8 @@
 import json
 import os
 
-from leapp.reporting import Remediation, Severity, create_report_from_error, create_report_from_deprecation
+from leapp.reporting import (_DEPRECATION_FLAGS, Groups, Remediation, Severity, create_report_from_error,
+                             create_report_from_deprecation)
```

Comment:
@fernflower separate please impoterted items 1 per line

Reply:
Should be fixed